### PR TITLE
🎨 Improve audio item layout

### PR DESCRIPTION
**Pull request by yanivshaked**
With current display, file name is cut, most of the real estate is used by the audio track symbol:

Suggesting to use 3 lines for the filename, moving the audio track next to the duration:

### CODE REVIEW

**Review by AlexV525 — APPROVED** [review]
LGTM but the icon seems not aligned pretty, can it be improved?

**Comment by yanivshaked** [comment]
Fixed:

File: lib/src/delegates/asset_picker_builder_delegate.dart [hunk 1]
```diff
@@ -1173,7 +1173,13 @@ class DefaultAssetPickerBuilderDelegate
             ),
           ),
         ),
-        const Align(alignment: AlignmentDirectional.bottomEnd, child: Icon(Icons.audiotrack)),
+        const Align(
+          alignment: AlignmentDirectional.bottomEnd,
+          child: Padding(
+            padding: EdgeInsetsDirectional.only(end: 4, bottom: 8),
```
Comment (AlexV525) [inline lib/src/delegates/asset_picker_builder_delegate.dart:1179]: Could we do this by setting the `alignment` property?

**Comment by AlexV525** [comment]
Also, a2dd6fc seems to be an invalid cherry-pick.

File: lib/src/delegates/asset_picker_builder_delegate.dart (hunk 1 above)
Comment (yanivshaked) [reply lib/src/delegates/asset_picker_builder_delegate.dart:1179]: If I use `alignment`, will RTL still be supported here?

Comment (AlexV525) [reply lib/src/delegates/asset_picker_builder_delegate.dart:1179]: `AlignmentDirectional` supports RTL.

**Comment by yanivshaked** [comment]
> Also, [a2dd6fc](https://github.com/fluttercandies/flutter_wechat_assets_picker/commit/a2dd6fcef7e1f73cf001c662b4105ca24dd94ed0) seems to be an invalid cherry-pick.

Fixed

File: lib/src/delegates/asset_picker_builder_delegate.dart (hunk 1 above)
Comment (yanivshaked) [reply lib/src/delegates/asset_picker_builder_delegate.dart:1179]: No good, using `Alignment` breaks support for RTL

File: lib/src/delegates/asset_picker_builder_delegate.dart [hunk 2]
```diff
@@ -1168,12 +1168,18 @@ class DefaultAssetPickerBuilderDelegate
...
           ),
         ),
-        const Center(child: Icon(Icons.audiotrack)),
+        const Align(
+          alignment: AlignmentDirectional.bottomEnd,
+          child: Padding(
+            padding: EdgeInsetsDirectional.only(end: 4, bottom: 8),
+            child: Icon(Icons.audiotrack),
+          ),
+        ),
```
Comment (AlexV525) [inline lib/src/delegates/asset_picker_builder_delegate.dart:1182]:
```suggestion
        const Align(
          alignment: AlignmentDirectional(0.9, 0.8),
          child: Icon(Icons.audiotrack),
        ),
```

**Review by AlexV525 — APPROVED** [review]
LGTM

**Pull request merged**